tests/cypress/e2e/for-teacher_page/class_page/class-survey.cy.js

```diff
@@ -0,0 +1,86 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+
+const surveyView = body => body.find("#survey")
+const classView = body => body.find(".view_class")
+describe('Class Survey View', () => {
+  it('Can respond to 1 questions', () => {
+    
+    loginForTeacher();
+    cy.wait(500);
```

Comment:
This wait is not really helping, could you remove it? We should really avoid adding hardcoded waits to our tests, they've given us headaches before.